distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -146,7 +146,7 @@ type Table
              table.select_columns (By_Column [column1, column2])
 
        Icon: select_column
-    select_columns : Column_Selector -> Boolean -> Problem_Behavior -> Table
+    select_columns : Vector | Column_Selector -> Boolean -> Problem_Behavior -> Table
```

Comment:
I know we are not consistent on this but in many places we used to do `Vector X`, can we?
```suggestion
    select_columns : Vector Text | Column_Selector -> Boolean -> Problem_Behavior -> Table
```